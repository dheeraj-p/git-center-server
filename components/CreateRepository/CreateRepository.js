export default function CreateRepository() {
  return (
    <div>
      <h2>Create Repository</h2>
      <p>Spaces are not allowed. Use '-' or '_' instead.</p>
      <div>
        <input
          type='text'
          name='repoName'
          class='form-control'
          placeholder='Repository Name'
        />
        <button type='button' class='btn btn-primary'>
          Create
        </button>
      </div>
    </div>
  );
}
