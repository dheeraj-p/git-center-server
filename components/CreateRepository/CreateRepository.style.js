import css from 'styled-jsx/css';

const CreateRepositoryStyles = css`
  .create-repository {
    border: 1px solid black;
    width: 100%;
    padding: 15px;
    margin: 15px;
  }

  .input-box {
    display: flex;
  }
`;

export default CreateRepositoryStyles;
