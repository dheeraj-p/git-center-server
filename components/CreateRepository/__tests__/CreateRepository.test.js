import { shallow } from 'enzyme';
import CreateRepository from '../CreateRepository';

describe('CreateRepository', () => {
  it('should render properly', () => {
    const wrapper = shallow(<CreateRepository />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
