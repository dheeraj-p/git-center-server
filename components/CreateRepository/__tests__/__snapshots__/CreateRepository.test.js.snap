// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateRepository should render properly 1`] = `
"<div className=\\"card mt-2\\">
  <div className=\\"card-body\\">
    <h5 className=\\"card-title\\">
      Create Repository
    </h5>
    <p className=\\"card-text text-muted\\">
      Spaces are not allowed. Use &#39;-&#39; or &#39;_&#39; instead.
    </p>
    <div className=\\"row no-gutters\\">
      <div className=\\"col-12 col-md-9 mb-2 mb-md-0\\">
        <input type=\\"text\\" name=\\"repoName\\" className=\\"form-control\\" placeholder=\\"Repository Name\\" />
      </div>
      <div className=\\"col-12 col-md-3\\">
        <button type=\\"button\\" className=\\"btn btn-success w-100 ml-md-2\\">
          Create
        </button>
      </div>
    </div>
  </div>
</div>"
`;
